dome9/data_source_dome9_aws_unified_onbording_update_version_stack_config_test.go

```diff
@@ -0,0 +1,42 @@
+package dome9
+
+import (
+	"testing"
+)
+
+
+func TestAccDataSourceAwsUnifiedOnboardingUpdateVersionStackConfigBasic(t *testing.T) {
+	//resourceTypeAndName, dataSourceTypeAndName, resourceName := method.GenerateRandomSourcesTypeAndName(resourcetype.AwsUnifiedOnboardingUpdateVersionStackConfig)
+	//awsTypeAndName, _, awsGeneratedName := method.GenerateRandomSourcesTypeAndName(resourcetype.AwsUnifiedOnboardingUpdateVersionStackConfig)
+	//
+	//
+	//return &schema.Resource{
+	//	Read: dataSourceAwsUnifiedOnboardingReadConfig,
+	//	Schema: map[string]*schema.Schema{
+	//		providerconst.OnboardingId: {
+	//			Type:     schema.TypeString,
+	//			Required: true,
+	//		},
+	//		providerconst.StackName: {
+	//			Type:     schema.TypeString,
+	//			Computed: true,
+	//		},
+	//		providerconst.Parameters: {
+	//			Type:     schema.TypeMap,
+	//			Computed: true,
+	//			Elem: &schema.Resource{
+	//				Schema: map[string]*schema.Schema{},
+	//			},
+	//		},
+	//		providerconst.IamCapabilities: {
+	//			Type:     schema.TypeList,
+	//			Computed: true,
+	//			Elem:     &schema.Schema{Type: schema.TypeString},
+	//		},
+	//		providerconst.TemplateUrl: {
+	//			Type:     schema.TypeString,
+	//			Computed: true,
+	//		},
+	//	},
+	//}
+}
```

Comment:
?????

Reply:
I write that onlt the resource test is Done but maybe its better to unstage this files for the next time